Mathlib/Algebra/BigOperators/Finprod.lean

```diff
@@ -626,6 +643,10 @@ theorem MonoidHom.map_finprod_mem (f : α → M) (g : M →* N) (hs : s.Finite)
     g (∏ᶠ j ∈ s, f j) = ∏ᶠ i ∈ s, g (f i) :=
   g.map_finprod_mem' (hs.inter_of_left _)
 
+theorem smul_finsum_mem {R M : Type*} [Monoid R] [AddCommMonoid M] [DistribMulAction R M] (c : R)
```

Comment:
Oops, I also added this in #23330

Reply:
Oh nice! I'll merge master and sort things out.